src/Client/Connection.cpp

```diff
@@ -551,6 +551,14 @@ void Connection::sendIgnoredPartUUIDs(const std::vector<UUID> & uuids)
     out->next();
 }
 
+
+void Connection::sendReadTaskResponse(const std::string & response)
+{
+    writeVarUInt(Protocol::Client::ReadTaskResponse, *out);
+    writeStringBinary(response, *out);
```

Comment:
Maybe we can also write version here. Just in case. And check that it is supported.

Reply:
Fixed